code/game/area/areas.dm

```diff
@@ -281,6 +281,24 @@ var/list/mob/living/forced_ambiance_list = new
 				H.AdjustWeakened(3)
 			to_chat(mob, "<span class='notice'>The sudden appearance of gravity makes you fall to the floor!</span>")
 
+/area/proc/throw_unbuckled_occupants(var/maxrange, var/speed, var/direction = null)
+	for(var/mob/M in src)
```

Comment:
This feels dangerous. Isn't there some consideration with looping on area contents weirdly expensive?